Vagrantfile

```diff
@@ -60,7 +60,7 @@
 ALAVETELI_FQDN = ENV['ALAVETELI_VAGRANT_FQDN'] || "alaveteli.10.10.10.30.xip.io"
 ALAVETELI_MEMORY = ENV['ALAVETELI_VAGRANT_MEMORY'] || 1536
 ALAVETELI_THEMES_DIR = ENV['ALAVETELI_THEMES_DIR'] || '../alaveteli-themes'
-ALAVETELI_OS = ENV['ALAVETELI_VAGRANT_OS'] || 'trusty64'
+ALAVETELI_OS = ENV['ALAVETELI_VAGRANT_OS'] || 'wheezy64'
```

Comment:
This doesn't update the changelog, which states that "trusty" is being made default.